### DISCUSSION TITLE
No block Win for a long period with 144TiB Miner

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#8240] [Storage Provider] [answered]
Please help me ~
my node f01474838 have no block since 2011/11/18
144T now
Everything works fine !!!
I cannot find any issue !!
Please have someone to help!

**Comment by weizihua** [comment]
You need to speed up your computing power package, or you can provide your logs for analysis

**Comment by Derek-zd** [comment]
miner 日志里 搜一下 "mined new block" ，看有没有出块日志

**Comment by seantw1974** [comment]
可以提供指令嗎？

Derek ***@***.***>於 2022年3月7日 週一，下午3:48寫道：

**Reply by Derek-zd** [reply]
tail -n20000000 miner.log |grep "mined new block" 
 miner.log 是miner 输出的日志

**Comment by seantw1974** [comment]
1. 之前沒有output miner.log , 現在也不可能去追前幾個月的log
2. 如果有mined new block 我就出塊了, 但很多人幫查,都沒有拿到出塊權

**Reply by Derek-zd** [reply]
看日志是为了确认是不是出块了，但是块丢了。
miner info 里有 Projected average block win rate 字段，看一下预计出块周期是多少。
miner log 是检查是否 有 "completed mineOne" 。

**Comment by Derek-zd** [comment]
没有log 就没法排查问题，保存log 非常重要

**Comment by LexLuthr** [comment, marked as answer]
@seantw1974 Did you figure out the problem? I see that total block win count is 3. What did you find in the logs?

**Reply by seantw1974** [reply]
In fact I still dont know the reason !
The old id miner cost me around 4 months without any mined blocked !! And then something damage my disk, I removed half sectors....
And from last time start to mined blocked ! This miner already stop to generate sectors!!
After stop the old miner , I start new miner id f01474838, as you can see , 58.25 TiB and have "2 mined blocked" alrady !!!! 
I  still believe something wrong with old one !! Maybe work again after I remove some (wrong) sectors!
I am now using new miner and work perfect !

**Reply by LexLuthr** [reply]
Okay. Can you post the logs from older miner with grep "isEligible". If your miner is not eligible then we need to check why.

**Reply by seantw1974** [reply]
Any way to resolve that ?

**Reply by seantw1974** [reply]
Sorry ...it's my fault !! I init my miner ...
Now I move back old backup ...it's ok now !

**Reply by seantw1974** [reply]
> Okay. Can you post the logs from older miner with grep "isEligible". If your miner is not eligible then we need to check why.

isEligible is True  no problem !!
Thank you!!

**Reply by lbj2004032** [reply]
Hello, I'd like to ask you how to solve this problem. I think I have the same problem as you

**Comment by LexLuthr** [comment]
I am archiving this discussion as the problem was fixed.
